buildscripts/bodo/conda-recipe/meta.yaml

```diff
@@ -51,11 +51,9 @@ requirements:
     - libcurl ~= 8.0
     - hdf5 >=1.14.3,<1.14.4.0a0=mpi_mpich_* # [not win]
     - h5py
-    - mpich 4.1.*=h* # [not win]
-    - msmpi # [win]
-    - vc # [win]
-    - vs2015_runtime # [win]
-    - vs2015_win-64 # [win]
+    - mpich 4.1.*=h*        # [not win]
+    - impi-devel=2021.13.1  # [win64]
+    - mpi4py ~= 4.0         # [win64]
```

Comment:
Can you move this to the other `mpi4py` so we can see the win and non-win together